### PR TITLE
MINOR: more powered by companies and updating apache logo

### PR DESCRIPTION
**Pull request by scott-confluent**
each new image has been optimized as well

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -444,15 +444,15 @@ tr:nth-child(odd) {
 }
 .logo-link {
     display: inline-block;
-    width: 220px;
-    height: 64.77px;
-    background: url("/images/logo.png");
```
Comment (junrao) [inline css/styles.css:449]: Should we remove the old logo image file?

File: css/styles.css [hunk 2]
```diff
@@ -444,15 +444,15 @@ tr:nth-child(odd) {
...
+    width: 117px;
+    height: 65px;
+    background: url("/logos/kafka_logo--simple.png");
+    background-size: auto 65px;
     background-repeat: no-repeat;
     /* small: 168 x 50 */
 }
 .page-dark .logo-link {
-    background: url('/logos/kafka-white.svg');
+    background: url('/logos/kafka_logo--white-simple.png.svg');
```
Comment (junrao) [inline css/styles.css]: kafka_logo--white-simple.png.svg => kafka_logo--white-simple.png ?

File: css/styles.css [hunk 3]
```diff
@@ -444,15 +444,15 @@ tr:nth-child(odd) {
...
-    background-size: 217px auto;
+    width: 117px;
+    height: 65px;
+    background: url("/logos/kafka_logo--simple.png");
+    background-size: auto 65px;
     background-repeat: no-repeat;
     /* small: 168 x 50 */
 }
 .page-dark .logo-link {
-    background: url('/logos/kafka-white.svg');
```
Comment (junrao) [inline css/styles.css:455]: Should we remove the old logo image file?

**Review by junrao — COMMENTED** [review]
@scott-confluent : Thanks for the PR.

Could we also link https://apache.org/logos/res/kafka/kafka_highres.png from http://kafka.apache.org/trademark ?

File: css/styles.css (hunk 3 above)
Comment (scott-confluent) [reply css/styles.css:455]: I've cleaned up all the "dark theme" stuff, which included any of the white logos that were in use.

File: css/styles.css (hunk 1 above)
Comment (scott-confluent) [reply css/styles.css:449]: removed

**Review by junrao — APPROVED** [review]
@scott-confluent : Thanks for the updated PR. LGTM

**Pull request merged**